flake.nix

```diff
@@ -27,6 +30,8 @@
           source-overrides = {
             passetto-client = inputs.passetto-hs + /client;
             passetto-core = inputs.passetto-hs + /core;
+            wai-middleware-prometheus = inputs.wai-middleware-prometheus + /wai-middleware-prometheus;
+            prometheus-client = inputs.wai-middleware-prometheus + /prometheus-client;
```

Comment:
You want to use https://github.com/juspay/prometheus-haskell/pull/3 and use prometheus exactly like `euler-hs` is being used, ie., via `imports` and `haskellFlakeProjectModules`.

https://github.com/nammayatri/shared-kernel/blob/12e660da6fcb5d09dc5a7b05b593a44c9583ee03/flake.nix#L24-L26

@aravindgopall @paulvictor Can we merge that PR?

Reply:
The PR is merged. @roman-bodavskiy You can just use https://github.com/juspay/prometheus-haskell/tree/more-proc-metrics like we use `euler-hs` here. No need to add any of these `source-overrides`.